Orange/widgets/data/owcontinuize.py

```diff
@@ -345,7 +345,10 @@ def normalize_by_span(var, data_or_dist, zero_based=True):
 
 def normalize_by_sd(var, data_or_dist):
     dist = _ensure_dist(var, data_or_dist)
-    mean, sd = dist.mean(), dist.standard_deviation()
+    if dist.shape[1] > 0:
+        mean, sd = dist.mean(), dist.standard_deviation()
+    else:
+        mean, sd = 0, 0
```

Comment:
The default `mean` should be 0, but the default `sd` should be 1.

You can set it to 0 and the lines below will, if you follow my suggestion for the previous commit, set it to 1. But reading the code will be easier if you immediately set it to 1.